src/ucp/core/ucp_ep.c

```diff
@@ -88,8 +88,6 @@ ucs_status_t ucp_ep_new(ucp_worker_h worker, uint64_t dest_uuid,
         ucs_queue_head_init(&ep->ext.stream->reqs);
         ep->ext.stream->ucp_ep       = ep;
```

Comment:
minor: now, when you removed two lines below, there is no need in such a big indent before `=`